### PR TITLE
Local CS fixer config

### PR DESCRIPTION
**Pull request by joshhornby**
@alexbowers Take a look and let me know what you think.

Don’t love the duplication between StyleCI and `.php_cs` but it will do for now.

https://github.com/astralapp/astral/issues/41

### CODE REVIEW

**Comment by joshhornby** [comment]
Note this will fail StyleCI checks, I think that should be a new PR fixing those.

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?php
+
+use Symfony\CS\Config\Config;
+use Symfony\CS\FixerInterface;
+use Symfony\CS\Finder\DefaultFinder;
+
+$fixers = [
```
Comment (alexbowers) [inline .php_cs]: Why so many predefined? We can define to use PSR2 in this as defaults, and then add or remove fixers as we see fit.

File: .php_cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    'short_echo_tag',
+    'short_tag',
+    'single_array_no_trailing_comma',
+    'single_blank_line_before_namespace',
+    'single_line_after_imports',
+    'single_quote',
+    'spaces_before_semicolon',
+    'spaces_cast',
+    'standardize_not_equal',
+    'ternary_spaces',
```
Comment (alexbowers) [inline .php_cs]: Is trailing commas in arrays setup for this?

```php
$arr = [
    'string',
    'string 2',
    'string 3',
];
```

Comment (alexbowers) [inline .php_cs]: Ah yes. Seen it above.

**Pull request merged**